client/app/config/index.js

```diff
@@ -57,7 +57,7 @@ function registerAll(context) {
     .map(context)
     .map(module => module.default);
 
-  return modules.map(f => f(ngModule));
+  return modules.filter(isFunction).map(f => f(ngModule));
```

Comment:
I'm assuming this is because of the utils.js file... let's just move `app/visualizations/chart/plotly/` content to `app/lib/visualizations`.

Reply:
I prefer to leave it - not especially because of `utils.js`, but rather for avoiding a sort of strange bugs - when just adding an empty file, app starts crashing, and it's a surprise for ones who doesn't know about this _autoloader_

Reply:
P.S. This also affects not just visualizations, but components, directives, services...

Reply:
Good point, let's keep it. 👍